src/main/java/org/polystat/Polystat.java

```diff
@@ -153,38 +154,35 @@ public Integer call() throws Exception {
     private Iterable<Result> scan(final Path src, final Path tmp) {
         final Func<String, XML> xmir = new Program(src, tmp);
         final Collection<Result> errors = new ArrayList<>(Polystat.ALL.length);
+        final String extension = ".eo";
         for (final Analysis analysis : Polystat.ALL) {
             try {
-                errors.addAll(new ListOf<>(analysis.errors(xmir, "\\Phi.test")));
+                for (final String filename : Objects.requireNonNull(src.toFile().list())) {
```

Comment:
@silverCase There is not point in `requireNonNull`. If required check for empty folder explicitly.